### PR TITLE
Add hip injury and knee injury cards

### PR DESCRIPTION
**Pull request by jvoros**

### CODE REVIEW

File: cards/hip-injuries-femoral-head-fracture/card.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+# Hip Injuries: Femoral Head Fracture
+
+![](image-1.png)
+
+## Xray Views
+
+- AP pelvis
+- Lateral hip
+- +/- Judet view of hip (45-degree oblique)
```
Comment (dsifford) [inline cards/hip-injuries-femoral-head-fracture/card.md]: Does marked decode html entities? If so, `+/-` should be converted to `	&plusmn;`

File: cards/hip-injuries-femoral-head-fracture/card.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+
+title: Hip Injuries - Femoral Head Fracture
```
Comment (dsifford) [inline cards/hip-injuries-femoral-head-fracture/card.md]: Can we choose a format and stick to it for these types?

Other cards that exist (I think ultrasound?) use colons. I prefer the dash as well (colons need to be escaped in yaml).

File: cards/knee-injuries-knee-dislocation/card.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+***Orthopedic vascular emergency!***
+
+Reduce knee and re-examine neurovascular status
+
+Consult vascular surgery for hard signs of vascular injury
+
+Measure Ankle-Brachial index(ABIs):
+
+- ABI ≥0.9: Monitor by serial exam
```
Comment (dsifford) [inline cards/knee-injuries-knee-dislocation/card.md]: `≥` should be `&ge;`

File: cards/knee-injuries-knee-dislocation/card.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+- ABI <0.9: Obtain CT angiography
```
Comment (dsifford) [inline cards/knee-injuries-knee-dislocation/card.md]: `<` should (probably) be `&lt;`

File: cards/knee-injuries-patella-fracture/card.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+**If Non-operative:**
+- Muscle strengthening education
+- Likely conversion to long leg cast or locking knee brace
+
+## Notes
+
+Operative Indications
+
+- Knee extensor function disruption
+- &gt; 2 mm articular displacement
```
Comment (dsifford) [inline cards/knee-injuries-patella-fracture/card.md]: Looks good here!

File: cards/hip-injuries-femoral-head-fracture/card.md (hunk 2 above)
Comment (jvoros) [inline cards/hip-injuries-femoral-head-fracture/card.md]: Good catch. Should be dashes, like on the knee cards.

File: cards/hip-injuries-femoral-head-fracture/card.md (hunk 1 above)
Comment (jvoros) [inline cards/hip-injuries-femoral-head-fracture/card.md]: Yes, marked will decode them. Will switch. There are some greater thans etc, that prolly need change

**Pull request merged**